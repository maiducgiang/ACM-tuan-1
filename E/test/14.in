100
264832303 9332561
330075225 401073028
13275648 321927337
847460576 45397289
391679702 188268271
97242084 331887479
372021403 353522526
212140778 19973572
994919207 569439355
303343255 879789811
350793159 373559808
292352883 967878009
9732891 773353116
489200777 953268043
555348703 713585694
188552484 306514202
524434644 613896853
152457518 433978909
245840006 13292155
33591407 197648093
774613559 129581715
421547717 975446806
617236846 561730593
991924237 601740464
654111915 74216735
751836085 234119999
571816143 161660987
56816591 842124487
156918786 212362133
228974679 57003488
618987170 737101004
681176771 985717057
82527758 963840889
645201635 351082493
51778379 988435512
338462225 128898416
368363074 692435973
234655394 162942124
282206950 818445968
128545813 737512630
104540357 10625348
310554403 377474514
944468760 665249807
47397277 628110341
283835636 92331135
122950752 744324425
237559178 399537408
257942011 697239718
429756902 553953277
884972254 279503155
530982905 800909339
287859545 248901000
833837546 234943367
598510220 787373049
905416611 940023907
796017229 307191386
667302292 9774365
78233691 385262508
400669173 202741818
889134436 104446338
537636161 350094497
839911950 632303576
899105874 245495935
59850557 761396641
59591986 29762180
116805301 923930512
815152167 740130287
498943424 324596377
976119746 820356903
463757086 419079449
256753192 436879166
871938288 610050310
22756846 295255783
136426499 892154680
294960958 405645349
389441812 253932398
288542815 830085895
979570058 108825626
777470116 16985964
136996248 583577752
816771076 798260107
190773306 344907001
198966038 572890441
984236745 764468888
812423648 954581495
693063853 311731814
776259841 33343921
968482650 587989600
222283631 85214031
11483066 206416672
199205184 644327325
290823232 944412947
284338904 970143058
744499265 312786154
723053463 637546800
959626133 816186002
136613340 439495395
473652718 348826152
173238430 567273195
722274746 118521338
