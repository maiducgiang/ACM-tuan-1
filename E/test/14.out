2
2
25
2
2
4
2
2
2
3
2
4
80
2
2
2
2
3
2
6
2
3
2
2
2
2
2
15
2
2
2
2
12
2
20
2
2
2
3
6
2
2
2
14
2
7
2
3
2
2
2
2
2
2
2
2
2
5
2
2
2
2
2
13
2
8
2
2
2
2
2
2
13
7
2
2
3
2
2
5
2
2
3
2
2
2
2
2
2
18
4
4
4
2
2
2
4
2
4
2
